src/Containers-OrderPreservingDictionary/CTOrderPreservingDictionary.class.st

```diff
@@ -1,6 +1,5 @@
 "
-By default, when CTOrderPreservingDictionaryionary raises an exception. 
-I'm a special kind of order-preserving dictionary that returns a configurable default value (nil by default) instead of raising an exception. 
+I'm a special kind of order-preserving dictionary that returns a configurable default value (nil by default) instead of raising an error. 
```

Comment:
It would be nice to explain what order-preserving means. Elements are stored in the order in which they were added? What happens if I "update" an entry? is it moved down (up?) in the order or it is left in the same order?

Imagine
```smalltalk
dict := CTOrderPreservingDictionary new. 
dict at: 1 put: 'first'.
dict at: 2 put: 'second'.
dict at: 1 put: 'another first'.
```
In which order is 'another first'? I think the comment should specify it (and put a piece of code as an example :)